### PR TITLE
STL-1958 Simplifying the Next Steps after registration

### PR DESCRIPTION
**Pull request by hamo225**
# Short Description
- We simplify the page outlining the next steps for users after a successful registration.

# Changes
- Created a new infoBox component success.
- changes to the unlockCodeSuccessPage for new designs
- added class to base.css to handle conditions with an icon and without
- added images for unlock code examples
- adds icons
- adds text to translations.js
- updated tests for unlockCodeSuccessPage
- added tests for the new successStepsInfoBox component
- added the new component to storybook

# Feedback
- are the tests conclusive for both the unlockCodeSuccessPage and the new component?

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/client/public/css/base.css [hunk 1]
```diff
@@ -2,580 +2,608 @@
...
+  }
+}
+
+@media (max-width: 425px) {
+  .steps_info_box_with_icon {
+    padding: 0 1px 1px;
+  }
+}
+
+.steps_info_box_without_icon {
```
Comment (SannyNguyenHung) [inline webapp/client/public/css/base.css:593]: Please don't add it here. Add those styles to the react page! :)

.InnerBox .steps_info_box_without_icon

**Review by SannyNguyenHung — CHANGES_REQUESTED** [review]
Looking good but some improvements needed :+1:

File: webapp/client/src/lib/translations.js [hunk 2]
```diff
@@ -345,19 +345,27 @@ const translations = {
     success: {
       "next-steps": {
         header: {
-          title: "Ihre Registrierung war erfolgreich!",
-          intro:
-            "Wir haben Ihren Antrag an Ihre Finanzverwaltung weitergeleitet. Sie können mit Ihrer Steuererklärung beginnen, sobald Sie Ihren Freischaltcode erhalten haben. Es kann bis zu zwei Wochen dauern, bis Sie Ihren Brief erhalten.",
+          title:
+            "Ihr Freischaltcode wurde bei Ihrem Finanzamt beantragt und wird Ihnen per Post zugeschickt.",
+          // intro:
```
[the remark this answers is not in the document]
Comment (hamo225) [reply webapp/client/src/lib/translations.js:350]: ok

File: webapp/client/public/css/base.css (hunk 1 above)
Comment (hamo225) [reply webapp/client/public/css/base.css:593]: much cleaner now thanks

File: webapp/client/src/components/successStepsInfoBox.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+import styled from "styled-components";
+import AnchorButton from "./AnchorButton";
+
+const Box = styled.div`
+  background-color: var(--white);
+  display: flex;
+  margin-top: var(--spacing-03);
+  border: 1px solid var(--beige-300);
+
+  @media (max-width: 425px) {
```
[the remark this answers is not in the document]
Comment (hamo225) [reply webapp/client/src/components/successStepsInfoBox.js:12]: speaking with design tomorrow regarding standard breakpoints.

**Review by SannyNguyenHung — APPROVED** [review]

**Pull request merged**